src/containers/HomePage.js

```diff
@@ -1,47 +1,73 @@
 import React, { Component } from 'react';
-import {
-  Button,
-  Container,
-  Header,
-  Icon,
-  Image,
-  Segment,
-} from 'semantic-ui-react';
+import {Button} from 'semantic-ui-react';
 import { Link } from 'react-router-dom';
 
-import logo_stacked from '../images/logo_stacked.svg';
 
 class HomePage extends Component {
   render() {
     return (
-      <div>
-        <Segment
-          inverted
-          textAlign='center'
-          style={{ minHeight: 500, padding: '1em 0em' }}
-          vertical
-          color='teal'
-        >
-          <Container text>
-          <br/>
-            <Image src={logo_stacked} size='small' inline />
-            <Header
-              as='h1'
-              content='Cliff Effects Tool'
-              inverted
-              style={{ fontSize: '4em', fontWeight: 'normal', marginBottom: 0, marginTop: '1em' }}
-            />
-            <Header
-              as='h2'
-              content='Intake a new client to begin.'
-              inverted
-              style={{ fontSize: '1.7em', fontWeight: 'normal' }}
-            />
-            <Link to="/intake"><Button size='big' inverted>Intake New Client<Icon name='right arrow'/></Button></Link>
-          </Container>
-        </Segment>
-
-        <Segment style={{ padding: '8em 0em' }} vertical textAlign='center' className={'spaceholder'}/>
+      <div id='HomePage'>
+
+        <div id='HomeContent'>
+
+          <div style={{
+          display:'inline-block'}}>
+
+            <h1 style={{
+            fontSize: '5em',
+            color: '#efefef',
+            marginBottom: '-15px'}}>
+              Cliff Effects Tool
+            </h1>
+
+            <h2 style={{
+            fontSize: '2em',
+            color: '#efefef',
+            textAlign: 'right',
+            marginTop: '-15px',
+            marginRight: '45px'}}>
```

Comment:
Generally prefer styling in the CSS file, but that needn't block this from going out.